workflows/epitopeprediction.nf

```diff
@@ -42,10 +42,11 @@ include { CHECK_REQUESTED_MODELS as CHECK_REQUESTED_MODELS_PEP }    from '../mod
 include { CHECK_REQUESTED_MODELS }                                  from '../modules/local/check_requested_models'
 include { SHOW_SUPPORTED_MODELS }                                   from '../modules/local/show_supported_models'
 
+include { VARIANT_SPLIT}                                            from '../modules/local/variant_split'
 include { SNPSIFT_SPLIT}                                            from '../modules/local/snpsift_split'
 include { CSVTK_SPLIT}                                              from '../modules/local/csvtk_split'
 
-include { FRED2_GENERATEPEPTIDES }                                  from '../modules/local/fred2_generatepeptides'
+include { GENERATE_PEPTIDES }                                       from '../modules/local/generate_peptides'
```

Comment:
Is there a nf-core convention for that?

Reply:
yes, in principle all nf-core modules would be called like `TOOL_SUBTOOL`. In the case of Epytope, because you have python scripts maybe it's more difficult to maintain like this, but could those modules then be called `EPYTOPE_GENERATE_PETIDES`, etc?